aip/general/0217.md

```diff
@@ -38,20 +38,43 @@ message ListBooksResponse {
 
 - The field **must** be a repeated string, and **should** be named
   `unreachable`.
-- The field **must** be set to the names of the resources which are the cause
-  of the issue, such as the parent or individual resources that could not be
-  reached. The objects listed as unreachable **may** be _parents_ (or higher
-  ancestors) rather than the individual resources being requested. For example,
-  if a location is unreachable, the location is listed.
-  - The response **must not** provide any other information about the issue,
-    such as error details or codes. To discover what the underlying issue is,
-    users **should** send a more specific request.
-  - The service **must** provide a way for the user to get an error with
-    additional information, and **should** allow the user to repeat the
-    original call with more restrictive parameters in order to do so.
-  - The resource names provided in this field **may** be heterogeneous. The
-    field **should** document what potential resources may be provided in this
-    field, and note that it might expand later.
+- The field **must** contain the resource names of the resources that are
+  unreachable or those that impede reaching the requested collection, such as
+  the parent resource of the collection that could not be reached.
+  - For example, if an entire location is unreachable, preventing access to the
+    localized collection of resources requested, the location resource is
+    included.
+- The field **must** contain _service-relative_ resource names, and **must not**
+  contain full resource names, resource URIs, or simple resource IDs. See
+  [AIP-122][aip-122] for definitions.
+  - For example, if a `Book` resource is unreachable, the _service-relative_
+    resource name `"shelves/scifi1/books/starwars4"` would appear in
+    `unreachable`, as opposed to the _full_ resource name
+    `"//library.googleapis.com/shelves/scifi1/books/starwars4"`, the
+    _parent-relative_ resource `"books/starwars4"`, the resource ID
+    `"starwars4"`, or the resource URI.
+- The response **must not** provide any other information about the issue(s)
+  that made the listed resources unreachable.
+  - For example, the response cannot contain an extra field with error reasons
+    for each `unreachable` entry.
+- The service **must** provide a way for the user to make a more specific
+  request and receive an error with additional information e.g. via a Standard
+  Get or a Standard List targeted at the unreachable collection parent.
+  - The service **must** also allow the user to repeat the original call with
+    more restrictive parameters.
+- The resource names that appear in `unreachable` **may** be heterogeneous.
+  - The `unreachable` field **should** document what potential resources could
```

Comment:
Possibly "The unreachable field definition" to make it clear that it's in the proto, not in the data itself? I may be being too picky :)

Reply:
Nah that's fine, happy to change it. Done.